cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -208,7 +212,91 @@ class CallSubqueryTest extends DocumentingTest {
+          #This can come in handy when doing large write operations, like batch updates or imports.
+          #To execute a subquery in separate transactions you add the modifier `IN TRANSACTIONS` after the subquery.""".stripMargin('#'))
+
+      implicit val csvFilesDir: File = createDir("csv-files")
+      val artistsCsvPath = new CsvFile("artists.csv").withContents(
+        Seq("1", "ABBA", "1992"),
+        Seq("2", "Roxette", "1986"),
+        Seq("3", "Europe", "1979"),
+        Seq("4", "The Cardigans", "1992")
+      )
+      p("""The following example imports a CSV file using the `LOAD CSV` clause, and
+          # creates nodes in separate transactions using `CALL {} IN TRANSACTIONS`
+          #
+          #.artists.csv
+          #[source]
+          #----
+          #include::csv-files/artists.csv[]
+          #----""".stripMargin('#'))
+      addQuery(
+        """LOAD CSV FROM '@csvFile' AS line
+          #CALL {
+          #  WITH line
+          #  CREATE (:Artist {name: line[1], year: toInteger(line[2])})
+          #} IN TRANSACTIONS OF 1 ROW""".stripMargin('#'),
+        assertions = ResultAssertions(r => r.isEmpty),
+        replacements = Seq(QueryTextReplacement("@csvFile", "file:///artists.csv", artistsCsvPath))
+      ) { resultTable() }
+      p("""As the size of the CSV file in this example is small, only a single separate transaction is
+          #started and committed.""".stripMargin('#'))
+      note(p("""`CALL { ... } IN TRANSACTIONS` is only allowed in <<query-transactions, implicit transactions>>"""))
+      section("Batching") {
+        p("""The amount of work to do in each separate transaction can be specified in terms of how many input rows
+            |to process before committing the current transaction and starting a new one.
+            |The number of input rows is set with the modifier `OF n ROWS` (or `ROW`).
+            |If omitted, the default batch size is 1000 rows.
+            |Here's the same example as above, but with one transaction every 2 input rows""".stripMargin)
+        addQuery(
+          """LOAD CSV FROM '@csvFile' AS line
+            #CALL {
+            #  WITH line
+            #  CREATE (:Artist {name: line[1], year: toInteger(line[2])})
+            #} IN TRANSACTIONS OF 2 ROWS""".stripMargin('#'),
+          assertions = ResultAssertions(r => r.isEmpty),
+          replacements = Seq(QueryTextReplacement("@csvFile", "file:///artists.csv", artistsCsvPath))
+        ) { resultTable() }
+        p("""The query now starts and commits two separate transactions""")
+        p(""". The first two executions of the subquery (for the first two input rows from `LOAD CSV`) take place in the first transaction.
+            |. The first transaction is then committed before proceeding.
+            |. The next two executions of the subquery (for the next two input rows) take place in a second transaction.
+            |. The second transaction is committed.""".stripMargin)
+      }
```

Comment:
Should we maybe include an example where `inputRows % batchSize != 0` just to clarify how we deplete the last batch?